Source/Core/Core/NetPlayClient.h

```diff
@@ -102,6 +107,8 @@ class NetPlayClient : public TraversalClientClient
   const PadMappingArray& GetPadMapping() const;
   const PadMappingArray& GetWiimoteMapping() const;
 
+  bool m_sync_save_data;
```

Comment:
This should be private

Reply:
If it needs to simply be only accessed, then add a const member function that acts as getter